tests/ApiTransformerTest.php

```diff
@@ -93,4 +93,62 @@ public function test_output_transforming()
 
         $this->assertEquals($expectedOutput, $transformedOutput);
     }
-}
\ No newline at end of file
+
+    public function test_the_datatype_is_nullable()
+    {
+        $this->transformer->setApiMapping([
+            'price' => ['newName' => 'price', 'dataType' => 'nullable|int']
+        ]);
+
+        $input = [
+            'price' => '100'
+        ];
+
+        $expectedOutput = [
+            'price' => 100
+        ];
+
+        $this->assertSame($expectedOutput, $this->transformer->transformOutput($input));
+
+        $input = [
+            'price' => 0
+        ];
+
+        $expectedOutput = [
```

Comment:
Why should zero become null. This is incorrect

Reply:
You can argument the same for for an empty array then?

I know it's a bit strange and i'm not agenst changing this either.

Reply:
I have just changed it up so that numeric numbers e.g. floats and integers doesn't return null.